components/playlist/browser/playlist_service.cc

```diff
@@ -319,8 +320,15 @@ void PlaylistService::NotifyPlaylistChanged(
     obs.OnPlaylistStatusChanged(params);
 
   // TODO(sko) Send proper events based on |params|
+  // for (auto& service_observer : service_observers_)
+  //   service_observer->OnEvent(mojom::PlaylistEvent::kUpdated);
+}
+
+void PlaylistService::NotifyPlaylistChanged(mojom::PlaylistEvent playlist_event,
+                                            const std::string& playlist_id) {
+  // TODO(sko) Send proper events based on |params|
   for (auto& service_observer : service_observers_)
-    service_observer->OnEvent(mojom::PlaylistEvent::kUpdated);
+    service_observer->OnEvent(playlist_event, playlist_id);
 }
```

Comment:
It looks like this method is called with the exaclty same arguments for the overloaded version above. I think removing this method and move the implementation into the above version would be better, so that we can keep the callsites simple.